backend/clubs/serializers.py

```diff
@@ -1143,6 +1143,21 @@ class Meta:
         fields = ("club", "role", "title", "active", "public")
 
 
+class UserUUIDSerializer(serializers.ModelSerializer):
+    """
+    Used to get the uuid of a user (for ICS Calendar export)
+    """
+
+    uuid = serializers.SerializerMethodField("get_uuid")
```

Comment:
@rohan-gupta8 A slightly more DRF way to do this is to use code like something below:

```python
username = serializers.CharField(source="person.username", read_only=True)
```

Also, if you want to make this a bit easier for the frontend, you can return an entire URL instead of just the UUID to make it easier for frontend to implement this. Also more robust IMO.

Reply:
How do I return a URL?

Reply:
https://github.com/pennlabs/penn-clubs/blob/master/backend/clubs/views.py#L616

^ Except instead of returning a QR code, just return a string instead. Ideally wrap the string in a JSON response, like `{url: "..."}`.

Reply:
Basically just return a string, where the string is a URL.